spec/associations/active_record_extensions_spec.rb

```diff
@@ -177,10 +177,10 @@ class Book < ActiveRecord::Base
           expect(school.city).to eq(city)
         end
 
-        it "returns nil when the belongs_to association class can't be autoloaded" do
+        it "doesn't raise any exception when the belongs_to association class can't be autoloaded" do
```

Comment:
According to [the commit](https://github.com/zilkey/active_hash/commit/209834d39f688597f0f85cebe69d30e8c6ae48c5) that added this test,
- It doesn't return `nil` in this case
- The purpose of the commit was to rescue a LoadError, so this test might have to check if it doesn't raise any exception, not its return value.